mne/datasets/eegbci/eegbci.py

```diff
@@ -92,8 +94,9 @@ def data_path(url, path=None, force_update=False, update_path=None, *, verbose=N
 
 @verbose
 def load_data(
-    subject,
+    subjects,
     runs,
+    *,
```

Comment:
For this I'd be tempted just to live with the misnomer `subject=[...]` rather than switch to `subjects=[...]` but the deprecation also won't be so terrible:
```
def load_data(
    subjects=None,
    runs=None,
    *,
    subject=None,
    ...
):
    if subject is not None:
        subjects = subjects
        warn(..., FutureWarning)
    del subject
```
And then you'll need to ensure that `subjects` and `runs` are not None, i.e., the user has actually supplied values for them (raise an error if None). Then once deprecation is complete we can remove the `=None` defaults safely.